subprojects/docs/src/docs/userguide/img/java-test-configurations.dot

```diff
@@ -0,0 +1,37 @@
+digraph javaPluginConfigurations {
```

Comment:
Shouldn't the digraph be named like the file? (just for clarity)

Reply:
I didn't change this because it doesn't seem to have an effect on anything and the Java Library Plugin image just had a name of "configurations". Probably worth changing though for clarity, as you say.